easybuild/easyconfigs/x/XZ/XZ-5.2.2-intel-2016a.eb

```diff
@@ -13,9 +13,6 @@ source_urls = ['http://tukaani.org/xz/']
 
 builddependencies = [
     ('Autotools', '20150215'),
-]
-
-dependencies = [
```

Comment:
@wpoely86 sure about this?

if so, do this in a separate PR for all XZ easyconfigs (incl. this one)